lib/order.rb

```diff
@@ -0,0 +1,28 @@
+class Order
+  attr_reader :order
+  def initialize(menu)
+    @menu = menu
+    @order = {}
+  end
+
+  def order_food(food, quantity)
```

Comment:
again `order.order_food` sounds weird - avoid mixing nouns and verbs in method names